Cargo.toml

```diff
@@ -12,10 +12,16 @@ categories = ["database-implementations", "data-structures"]
 
 [dependencies]
 tantivy = "0.12"
-sled = { version = "0.31", path = "../sled" }
-bincode = "1"
+sled = { version = "0.31"}
+bincode = { version =  "1", optional = true }
 serde = { version = "1", features = ["derive"] }
 thiserror = "1"
 rayon = "1"
 pallet-macros = { path = "./pallet-macros", version = "0.4" }
 tempfile = "3.1"
+serde_cbor ={ version =  "0.11.1", optional = true }
+
+[features]
+default = []
+serde-bincode = ["bincode"]
+cbor = ["serde_cbor"]
```

Comment:
Unless you anticipate needing to pull in additional crates individually for either the `bincode` or `cbor` de/serialization, I'd say just remove the `serde-bincode` and `cbor` feature flags (you can just use the crate name as a feature flag). I'd also like to set `default = ["bincode"]`.

Reply:
I've had problem selecting cbor only in my project as the feature gate does not seem to work.

My configuration was 
```toml
[dependencies.pallet]
path = "../pallet"
default-features = false
features = ["serde_cbor"]
```